src/components/Dashboard/Form/FormSteps.js

```diff
@@ -17,7 +21,7 @@ const onSubmit = async values => {
 const FormSteps = () => (
   <React.Fragment>
     <DocumentForm
-      initialValues={{ schoolType: "Szkoła Podstawowa", profession: false }}
+      initialValues={commonData}
```

Comment:
I am not sure if that initialValues should be completely changed or if there is no better way to pass commonData in this form. Of course these "schoolType" and "profession" were only examples, however, initial values refers to values for inputs so for instance if we have dropdown - initial value in that case could be "Szkoła Podstawowa".

Reply:
I think we should prepare some way do populate inputs with data from fixtures (in the future with data from database). Maybe it would be better to insert another variable dedicated to dropdowns data.